### DISCUSSION TITLE
Updating Win 10 to Win 11 problem

### DISCUSSION THREAD

**Discussion by ozilwg** [AveYo/MediaCreationTool.bat#180] [General]
I wanted to do the Win 10 to Win 11 upgrades without losing any data. I used mediacreationtool and the installer for windows server was done automatically, I do not know if this is the problem, but in the final stage of the installation, after reboots, such an error pops up
**0x8007042b 0x4001e second_boot pre_oobe**

![image](https://user-images.githubusercontent.com/94186149/184373266-ae9e1343-5a0e-4f4d-a042-b11b50619127.png)

**Comment by AveYo** [comment]
that's a known windows update error.  
it's unrelated to the script.
check `C:\$WINDOWS.~BT\Sources\Panther` folder for logs with more details about the error (a driver or program fails to migrate)

**Reply by ozilwg** [reply]
Today I tried to reinstall this after downloading the iso for a couple of hours it stood in the preparation area. Which file to check?

![image](https://user-images.githubusercontent.com/94186149/184450588-09c19a5a-6335-4871-8479-29e75b756145.png)

**Reply by AveYo** [reply]
All ;) but specifically setuperr and setupact

**Reply by ozilwg** [reply]
I will restart the installation and add these 2 files
Can I delete these 3 folders before installing from iso?
![image](https://user-images.githubusercontent.com/94186149/184461487-74e7f7fb-ad96-4c53-a0e1-d8019b97fa45.png)
Because checking is endless
![image](https://user-images.githubusercontent.com/94186149/184461546-fdd962b6-e1bb-4396-87df-17f9938b8dcd.png)

**Comment by AveYo** [comment]
or use [windowss_update_refresh.bat](https://github.com/AveYo/MediaCreationTool.bat/blob/main/bypass11/windows_update_refresh.bat)  

what build are you trying to install?  
what method / script to bypass?

**Reply by ozilwg** [reply]
Win11_Polish_x64v1
Yes, with script to bypass
[setuperr.log](https://github.com/AveYo/MediaCreationTool.bat/files/9331635/setuperr.log)

**Comment by AveYo** [comment]
Errors you get make no sense.
It's like bad RAM / antivirus denying operations. Do you use a custom AV?
And the issues migrating paint and Mfplat.dll package is also weird. Is your 10 a N version with media feature pack? 
All I have is questions, because you've only shared setperr, that is missing the big picture context.

In any case, you need to do an in-place upgrade with your current windows 10 build to repair whatever is wrong.

**Reply by ozilwg** [reply]
Setupact is 220 mb. I have an internet security eset on which the license has expired. Windows 10 Pro.
I have to download the latest iso file Win 10 and install it from the system level without removing danch and then try again with iso Win 11 will it work?

**Reply by AveYo** [reply]
that's the best bet.  
you should remove eset beforehand (temporarily, until you get 11 up and running)

**Reply by ozilwg** [reply]
Could you please check the log from this update? Because I have the same error all the time
[WindowsUpdate.log](https://github.com/AveYo/MediaCreationTool.bat/files/9600447/WindowsUpdate.log)

**Reply by AveYo** [reply]
Sorry, I don't have the time to comb trough 50K outdated lines atm.
Try clearing previous entries to have a current, fresh log.
[bypass11/windows_update_refresh.bat](https://github.com/AveYo/MediaCreationTool.bat/blob/main/bypass11/windows_update_refresh.bat) can help with that

**Reply by ozilwg** [reply]
After cleaning and trying to install again, do a log of this? Get-WindowsUpdateLog

**Reply by ozilwg** [reply]
[WindowsUpdate.log](https://github.com/AveYo/MediaCreationTool.bat/files/9605543/WindowsUpdate.log)

**Reply by AveYo** [reply]
New log does not state any cause.
Have you removed any built-in system apps? 

Some generic points:
- make sure your clock is synchronized online (enable in settings) 
- try in a cmd (admin):
```bat
reg delete "HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate" /f
reg delete "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\WindowsUpdate\Auto Update" /f
sc.exe config wlidsvc start=demand
```
- run setup from a same version iso to repair whatever is broken

**Reply by ozilwg** [reply]
Yes, I deleted skype applications and maybe something else but it's a very long time ago

Date and time synchronization turned on

Earlier, I downloaded windows 10 iso to try to fix it and the same error was at the end of the installation after 94%, I think there is undoing changes

**Reply by AveYo** [reply]
Probably fails after reboot when migrating data so it's not logged in windows.
"Debloating" is very hard to diagnose, even harder to fix.
These might help, enter in cmd (admin):
```bat
reg delete HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Appx\AppxAllUserStore\Deprovisioned /f
reg delete HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Appx\AppxAllUserStore\EndOfLife /f
```
Try to grab everything in `C:\$WINDOWS.~BT\Sources\Panther` and provide it.

**Reply by ozilwg** [reply]
I have this zipped folder to send after yesterday's iso update attempt or after doing what you wrote?
Here I have a zip after I try to update from iso for the first time
[Panther.zip](https://github.com/AveYo/MediaCreationTool.bat/files/9612325/Panther.zip)

**Reply by ozilwg** [reply]
Everything works. It was enough to turn the windows defender back on with this program
`
https://www.sordum.org/9480/defender-control-v2-1/
`
